web-services/common/src/test/java/datawave/webservice/common/cache/SharedCacheCoordinatorTest.java

```diff
@@ -108,7 +108,7 @@ public void stateChanged(CuratorFramework client, ConnectionState newState) {
             for (int i = 0; i < 50; ++i) {
                 if (count[0] == newCount)
                     break;
-                Thread.sleep(200L);
+                Thread.sleep(1000);
```

Comment:
Why is this sleep statement increasing by 5x? Please retain the long formatted number like `1000L`